account_sdk/src/suppliers/devnet.rs

```diff
@@ -0,0 +1,53 @@
+use starknet::{
+    macros::felt,
+    providers::{jsonrpc::HttpTransport, JsonRpcClient},
+};
+use url::Url;
+
+use super::{PrefundedClientSupplier, RpcClientSupplier};
+
+use super::{AccountData, PredeployedClientSupplier, PredeployedContract};
+
+// starknet-devnet-rs
+// cargo run -- --port <port> --seed 0
+#[derive(Debug, Clone)]
+pub struct DevnetSupplier {
+    pub port: u16,
+}
+
+impl PrefundedClientSupplier for DevnetSupplier {
+    fn prefunded_account(&self) -> AccountData {
+        AccountData {
+            account_address: felt!(
+                "0x64b48806902a367c8598f4f95c305e8c1a1acba5f082d294a43793113115691"
+            ),
+            private_key: felt!("0x71d7bb07b9a64f6f78ac4c816aff4da9"),
+            public_key: felt!("0x39d9e6ce352ad4530a0ef5d5a18fd3303c3606a7fa6ac5b620020ad681cc33b"),
+        }
+    }
+}
+
+impl PredeployedClientSupplier for DevnetSupplier {
+    
+    fn predeployed_fee_token(&self) -> PredeployedContract {
```

Comment:
these addresses are constant across all networks. no need to have a generic impl. global consts is fine